app/routers/event.py

```diff
@@ -160,6 +170,31 @@ async def create_new_event(
     )
 
 
+def process_image(
+    img: bytes,
+    event_id: int,
+    img_height: int = IMAGE_HEIGHT,
+) -> str:
+    """Resized and saves picture without exif (to avoid malicious date))
+    according to required height and keep aspect ratio"""
+    try:
+        image = Image.open(io.BytesIO(img))
+    except IOError:
+        error_message = "The uploaded file is not a valid image"
+        logger.exception(error_message)
+        return
+    width, height = image.size
+    height_to_req_height = img_height / float(height)
+    new_width = int(float(width) * float(height_to_req_height))
+    resized = image.resize((new_width, img_height), Image.ANTIALIAS)
+    file_name = f"{event_id}{PICTURE_EXTENSION}"
+    image_data = list(resized.getdata())
+    image_without_exif = Image.new(resized.mode, resized.size)
+    image_without_exif.putdata(image_data)
+    image_without_exif.save(f"{MEDIA_PATH}/{file_name}")
```

Comment:
For security reasons, there should be an `UPLOAD_PATH` that is separate from the `MEDIA_PATH`.

You can set it in the configuration file.

Reply:
Where should I put the upload path? should it be a sub directory of media or an entirely new directory?

Reply:
I don't want to be delayed (my 3rd PR woohoo!) so I created a directory under "app" as I saw several folders were created like this.
Easy to change if you just say the word